### PR TITLE
`treehouses led labourday` (fixes #1534)

### LINKED ISSUE #1534: `treehouses led <festivity>`?

### PR DESCRIPTION
**Pull request by koxuan**
fixes #1534

new led pattern based on Labour Day that occurs on May 1 internationally.

5 slow green blinks to signify 5 long working days.
2 fast red blinks to signify 2 fast rest days.

This cycle runs for 2 times.

### CODE REVIEW

File: modules/led.sh [hunk 1]
```diff
@@ -109,6 +109,14 @@ function led {
       echo "this will happen 3 times"
       dragonboat > "$LOGFILE"
       ;;
+      labourday)
+      checkroot
+      echo "leds are set to labourday mode."
+      echo "Look at your RPi leds, both leds will be in this pattern..."
+      echo "Green LED: flashing 5 times with a decreasing frequency"
```
Comment (fankaiyuan) [inline modules/led.sh:116]: not seeing a decreasing frequency pattern based on your code

File: tests/led/labourday.bats [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bats
+load ../test-helper
+
+@test "$clinom led dragonboat" {
```
Comment (fankaiyuan) [inline tests/led/labourday.bats:4]: labourday instead of dragonboat

File: README.md [hunk 3]
```diff
@@ -72,8 +72,8 @@ sshtunnel [add|remove|list|active|check]  helps adding sshtunnels
 led [green|red] [mode]                    sets the led mode
     [newyear|lunarnewyear|valentine]
     [carnival|lantern|stpatricks|easter]
-    [eid|dragonboat|independenceday|onam]
-    [diwali|thanksgiving|christmas]
+    [eid|dragonboat|labourday|independenceday]
```
Comment (fankaiyuan) [inline README.md:75]: I think days are sorted by the calendar dates

**Comment by koxuan** [comment]
> 
> 
> This is executing 2 times, then green led flashes continuously. Although, it not specified in the code.
> Also, I could not see any decreasing frequency.
> 
> `bats labourday.bats` tests for dragonboat instead of labourday.

the green led flashing continuosly is not appearing on my rpi. Is there anything I can do to replicate this?

**Pull request merged**